Soccar [Library]/src/nl/soccar/library/enumeration/ThrottleAction.java

```diff
@@ -7,6 +7,31 @@
  */
 public enum ThrottleAction {
     
-    IDLE, ACCELERATE, REVERSE
+    IDLE(0), 
+    ACCELERATE(1), 
+    REVERSE(2);
+    
+    private final int id;
+
+    private ThrottleAction(int id) {
+        this.id = id;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static ThrottleAction parse(int id) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)